examples/dodge-the-creeps/rust/src/player.rs

```diff
@@ -101,10 +101,10 @@ impl GodotExt for Player {
         }
 
         let change = velocity * delta as f32;
-        let position = self.base.get_global_position().inner() + change;
+        let position = self.base.get_global_position() + change;
         let position = Vector2::new(
-            position.x.max(0.0).min(self.screen_size.inner().x),
-            position.y.max(0.0).min(self.screen_size.inner().y),
+            position.x().clamp(0.0, self.screen_size.x()),
+            position.y().clamp(0.0, self.screen_size.y()),
```

Comment:
This is probably the only change that's still relevant now.

Reply:
nice spot.  i merged the suggested change